src/migrate.rs

```diff
@@ -1,4 +1,20 @@
-//! Migrate default struct between two schemas
+//! Migrate pattern implements methods to change the storage representation of a struct.
+//!
```

Comment:
[Suggestion] when describing a new component start a new line

Reply:
I didn't understand your suggestion, can you share what you expect?

Reply:
Yeah, for example 

```
//! the contract state from the old schema. The [`on_migrate`][`MigrateHook::on_migrate`]
//! method takes this state and replaces it with the new schema.
```

Recommendation would be to format something like this:
```
//! the contract state from the old schema. 
//! The [`on_migrate`][`MigrateHook::on_migrate`] method takes this state and replaces it with the new schema.
```